rt_dashboard/history.py

```diff
@@ -25,12 +24,21 @@ def task_tooltip(t):
 
 
 def get_history_context():
-    tz = pytz.timezone(settings.TIMEZONE)
-    tasks = finished_task_registry.get_tasks() + failed_task_registry.get_tasks()
-    start = utcnow() - datetime.timedelta(days=2)
-    tasks = [t for t in tasks if t.started_at > start]
+    max_finished_tasks = 5000
```

Comment:
@i-trofimtschuk Would this work for yf, as yf seems to have more tasks per day than that?

Reply:
we have ~90k-100k per week so more like 15000

Reply:
I had the impression that ston is close to the limit of what the system can handle, that's why I chose this number. I'm pretty sure that yf just has too many tasks per day to display them all. But let's deploy this for now and then we can see whether there is room for displaying more tasks.

Reply:
I agree with that. In any case 5000 tasks being displayed is better than no tasks as is currently the case at yf